docs/drafts/upgrade-support.md

```diff
@@ -0,0 +1,54 @@
+# Upgrade support
+
+This document explains how OLM 1.0 handles upgrades.
+
+OLM 1.0 introduces a simplified UX for package authors and package admins to implicitly define upgrade edges via [Semantic Versioning](https://semver.org/).
+
+It also introduces an API to enable independently verified upgrades and downgrades.
+
+## Upgrade constraint semantics
+
+As of Operator Controller version 0.x, OLM 1.0 supports the following upgrade constraint semantics:
```

Comment:
Just to clarify this comment yesterday, I was sure what version of the Operator Controller introduced this feature, so we could update it to something like: "Operator Controller version 0.11.0" or something.

Reply:
Ahh, missted that. I'll update that

Reply:
It is tricky, but I think you came up with an elegant solution.